test/readBuffer.test.js

```diff
@@ -0,0 +1,49 @@
+const stream = require('stream');
+const readBuffer = require('../dist/readBuffer');
+
+function dataReadTest(done) {
+  expect.assertions(3);
+  let eventCount = 0;
+  function read() {
+    eventCount += 1;
+    if (eventCount <= 8) {
+      return this.push(Buffer.from(eventCount.toString()));
+    }
+    return this.push(null);
+  }
+  const mockEventStream = new stream.Readable({
+    objectMode: true,
+    read,
+  });
+  function cb(err, data) {
+    expect(err).toBe(null);
+    expect(data.length).toBe(8);
+    expect(String.fromCharCode(data[0])).toBe('1');
+    done();
+  }
+  readBuffer.default(mockEventStream, 8, cb);
+}
+
+test('data is read', dataReadTest);
```

Comment:
Just curious, why'd you define the function above?

Reply:
I thought it was necessary to pass the linter

Reply:
Switched it back to the normal way.